Task_1/fets_challenge/create_test_csv.py

```diff
@@ -0,0 +1,26 @@
+import os
+import csv
+
+# This script generates a CSV file named 'final_test.csv' in the current directory.
+# The CSV file has two columns: 'Partition_ID' and 'Subject_ID'.
+# 'Partition_ID' is set to -1 for all rows.
+# 'Subject_ID' is populated with the names of folders in the specified directory.
+# This is intended to replicate the format of 'validation.csv' in the FeTS Challenge repository.
+# The generated CSV file can be used to list all test samples in your data path.
+
+# Specify the directory containing the test data folders
+data_dir = "/home/locolinux2/datasets/RSNA_ASNR_MICCAI_BraTS2021_TestingData"
+
+dir_list = [d for d in os.listdir(data_dir) if os.path.isdir(os.path.join(data_dir, d))]
+
+output_csv =  '/home/locolinux2/.local/workspace/final_test.csv'

```

Comment:
Might make sense to make these as parameters and have that as input, otherwise it will break for anyone not having this exact directory/file-system structure.